### PR TITLE
Bootstrap Jenkins 2

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Review by blackheaven — APPROVED** [review]

File: jenkins.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code plugin\n\n"
+  numExecutors: 1
```
Comment (rouazana) [inline jenkins.yaml:3]: why do you have this in jenkins.yaml and executors.groovy?

File: Dockerfile [hunk 2]
```diff
@@ -3,6 +3,9 @@ FROM jenkins/jenkins:2.168
 USER jenkins
 
 COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
+COPY jenkins.yaml /usr/share/jenkins/ref/jenkins.yaml
+
+ENV CASC_JENKINS_CONFIG=/usr/share/jenkins/ref/
```
Comment (rouazana) [inline Dockerfile:10]: what does mean CASC?

Comment (rouazana) [reply Dockerfile:10]: ohh configuration as code I guess, strange way to write this acronym

File: jenkins.yaml (hunk 1 above)
Comment (aduprat) [reply jenkins.yaml:3]: Good catch, I will rewrite the history: `executors.groovy` is not used.

File: Dockerfile (hunk 2 above)
Comment (aduprat) [reply Dockerfile:10]: That's it, I've no hand on such environment variable.
BTW, I will add a comment for that.

File: jenkins.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+jenkins:
```
Comment (mbaechler) [inline jenkins.yaml:1]: Woot !

**Review by mbaechler — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by trantienduchn** [comment]
read it

**Comment by aduprat** [comment]
`push -f` due to comments

**Review by rouazana — APPROVED** [review]

**Review by mbaechler — APPROVED** [review]

**Pull request merged**